core/src/main/java/org/mapfish/print/servlet/MapPrinterServlet.java

```diff
@@ -1022,6 +1022,10 @@ private boolean checkReferer(
             return true;
         }
         final String referer = request.getHeader("referer");
+        if (referer == null) {
+            LOGGER.debug("Null referer");
+            return false;
```

Comment:
Not sure that a good idea, it can make debugging harder...
Why not change it to localhost that we can permit it with the config?

Reply:
OK, why not